src/pages/future/index.tsx

```diff
@@ -31,7 +31,7 @@ const FutureLaunches: React.FC = () => {
   );
 
   return (
-    <div className="flex flex-col justify-center items-center text-center">
+    <div className="flex flex-col justify-center items-center text-center mt-20 overflow-hidden ">
       <h2 className="mb-12">Future Launches</h2>
```

Comment:
I've added an overflow hidden while waiting for the refacto of the table displayed on mobile viewpart because it was messing the font size, relative to the width of the view.